File: src/main/java/com/vaadin/demo/component/cookieconsent/CookieConsentBasic.java
package com.vaadin.demo.component.cookieconsent;

import com.vaadin.demo.DemoExporter;
import com.vaadin.flow.component.cookieconsent.CookieConsent;
import com.vaadin.flow.component.html.Div;
import com.vaadin.flow.router.Route;

@Route("cookie-consent-basic")
public class CookieConsentBasic extends Div {

    public CookieConsentBasic() {
        // tag::snippet[]
        CookieConsent cookieConsent = new CookieConsent();

        add(cookieConsent);
        // end::snippet[]
    }

    public static class Exporter extends DemoExporter<CookieConsentBasic> { // hidden-source-line
    } // hidden-source-line
}

File: frontend/demo/component/cookieconsent/cookie-consent-basic.ts
import 'Frontend/demo/init'; // hidden-source-line
import './example-cleanup'; // hidden-source-line
import { html, LitElement } from 'lit';
import { customElement } from 'lit/decorators.js';
import '@vaadin/vaadin-cookie-consent/vaadin-cookie-consent';

@customElement('cookie-consent-basic')
export class Example extends LitElement {
  render() {
    return html`
      <!-- tag::snippet[] -->
      <vaadin-cookie-consent></vaadin-cookie-consent>
      <!-- end::snippet[] -->
    `;
  }
}
